src/controllers/chatTribes.ts

```diff
@@ -635,7 +640,7 @@ export async function replayChatHistory(chat, contact, ownerRecord) {
         includeStatus
       )
 
-      msg = await decryptMessage(msg, chat)
+      msg = await decryptMessage({ full: msg }, chat) // did this work before? (expected 2 arguments, but got 1)
```

Comment:
I'm confused why it's giving that error to you isn't `msg = await decryptMessage(msg, chat)` passing two arguments?